livre/Eq/Config.cpp

```diff
@@ -253,9 +279,13 @@ bool Config::init()
     subscriber->registerHandler( zeq::hbp::EVENT_LOOKUPTABLE1D,
                                  boost::bind( &detail::Config::onLookupTable1D,
                                               _impl, _1 ));
-
-    SubscriberPtr rcSubscriber( new zeq::Subscriber(
-                                    lunchbox::URI( "restcmd://" )));
+#ifdef LIVRE_USE_RESTCONNECTOR
+    std::string subscriberSchema = _impl->framedata.getNetworkParameters()->zeqSchema
+                                   + SUBSCRIBER_SCHEMA_SUFFIX;
+    SubscriberPtr rcSubscriber( new zeq::Subscriber( lunchbox::URI( subscriberSchema ) ) );
+#else
+    SubscriberPtr rcSubscriber( new zeq::Subscriber( lunchbox::URI( "restcmd://" ) ) );
```

Comment:
Yes. We want to keep the possibility to control the application only with zeq. I renamed "rest" and "rc" to "vws" so it is more general.